worlds/hk/__init__.py

```diff
@@ -321,6 +321,9 @@ def __init__(self, name, advancement, code, type, player: int = None):
         if name == "Mimic_Grub":
             self.trap = True
 
+        if type in ("Grub", "Charm", "DreamWarrior", "Root", "Egg"):
```

Comment:
Some charms are advancement items and should be balanced, though. Kono gave a list in the GH issue on my page. I can get you the charm IDs if that's what you need.

Reply:
If not, it's actually names, not IDs

Reply:
Here's the list of names: https://github.com/Ijwu/Archipelago.HollowKnight/issues/46

Reply:
charm list added in 6a715ff. I pulled the names directly out of ExtractedData.py if that is not correct then.. whooops

Reply:
Once berserker approves that commit, I'll approve the PR.

Reply:
Berserker says "yep looks good" in voice chat.